modules/aws-outposts-requirements-limitations.adoc

```diff
@@ -30,4 +36,8 @@ The Outpost has edge compute nodes, while the VPC has cloud-based compute nodes.
 The cloud-based AWS Elastic Block volume cannot attach to Outpost edge compute nodes, and the Outpost volumes cannot attach to cloud-based compute nodes.
 +
 As a result, you cannot use CSI snapshots to migrate applications that use persistent storage from cloud-based compute nodes to edge compute nodes or directly use the original persistent volume.
-To migrate persistent storage data for applications, you must perform a manual backup and restore operation.
\ No newline at end of file
+To migrate persistent storage data for applications, you must perform a manual backup and restore operation.
+
+* AWS Outposts does not support AWS Network Load Balancers or AWS Classic Load Balancers.
+To manages Ingress objects for your edge compute resources, you must install the AWS Load Balancer Operator so that you can use AWS Application Load Balancers in the AWS Outposts environment.
```

Comment:
It seems to me that there is no transition between "Outposts doesn't support NLB" and "Install ALBO to manage Ingress resources". May be something like - "Application Load Balancer is the only LB type supported on AWS Outposts. To be able to provision it you must 1) use Ingress resource and 2) install ALBO ".